File: Project_API/src/main/java/com/library/model/Borrow.java
package com.library.model;
///////////////////////////////////////////////////////////////////////////////////////////////////////    
//  FILE : Borrow.java
//  AUTHOR : Pranav Sehgal <PranavSehgalCS>
//  DESCRIPTION: Is a public model to encapsulate borrow data
///////////////////////////////////////////////////////////////////////////////////////////////////////
public class Borrow {
    private int bookId;
    private String borrDate;
    private String accName;
    private String bookName;

    public Borrow(String accName, String borrDate,int bookId, String bookName){
        this.bookId = bookId;
        this.accName = accName;
        this.borrDate = borrDate;
        this.bookName = bookName;
    }

    public int getBookId(){
        return bookId;
    }
    public String getBorrDate(){
        return borrDate;
    }
    public String getAccName(){
        return accName;
    }
    public String getBookName(){
        return bookName;
    }
}

File: Project_UI/angular-workspace/src/app/model/Borrow.ts
///////////////////////////////////////////////////////////////////////////////////////////////////////
//  FILE : Borrow.ts
//  AUTHOR : Pranav Sehgal <PranavSehgalCS>
//  DESCRIPTION: Is a model with constructor to encapsulate borrow data 
///////////////////////////////////////////////////////////////////////////////////////////////////////
export class Borrow{
    public accName:string = "";
    public borrDate:string = "";
    public bookId:number = -1;
    public bookName:string = "";

    public constructor(accName:string, borrDate:string, bookId:number, bookName:string) {
        this.accName = accName;
        this.borrDate = borrDate;
        this.bookId = bookId;
        this.bookName = bookName;
    }
}